Dockerfile

```
@@ -0,0 +1,46 @@
+# use the official Bun image
+# see all versions at https://hub.docker.com/r/oven/bun/tags
+FROM oven/bun:1 AS base
+WORKDIR /usr/src/app
+
+RUN apt update
+
+RUN apt install -y \
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)